.github/workflows/main.yml

```diff
@@ -37,42 +41,32 @@ jobs:
           export VERSION=$(./mvnw -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
           echo "version=${VERSION}" >> $GITHUB_OUTPUT
 
-      # docker images
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v3
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
-
-      - name: Cache Docker layers
-        uses: actions/cache@v4
+      - name: Upload jar
+        uses: actions/upload-artifact@v4
         with:
-          path: /tmp/.buildx-cache
-          key: ${{ runner.os }}-buildx-${{ github.sha }}
-          restore-keys: |
-            ${{ runner.os }}-buildx-
+          name: kafbat-ui-${{ steps.build.outputs.version }}
+          path: api/target/api-${{ steps.build.outputs.version }}.jar
+          retention-days: 7
```

Comment:
let's do 1, the jars are pretty fat to hold in the cache for too long

Reply:
Sure, fixed in [76ed720](https://github.com/giom-l/kafka-ui/commit/76ed720828c409376ba55785e7e4f286cb6ba2aa)